core/src/client/async_client/mod.rs

```diff
@@ -339,7 +339,7 @@ impl ClientT for Client {
 			Err(_) => return Err(self.read_error_from_backend().await),
 		};
 
-		rx_log_from_json(&Response::new(&json_value, id), self.max_log_length);
+		rx_log_from_json(&Response::new(PartialResponse::Result(json_value.clone()), id), self.max_log_length);
```

Comment:
I wonder what the cost of cloning is here?

Reply:
we can provide a some borrowing here, I didn't bother just did naive thing :P